src/main/java/core/basesyntax/service/impl/CalculatorImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Calculator;
+import core.basesyntax.service.OperationService;
+import core.basesyntax.strategy.OperationServiceStrategy;
+import java.util.List;
+
+public class CalculatorImpl implements Calculator {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private OperationServiceStrategy operationServiceStrategy = new OperationServiceStrategy();

```

Comment:
let's get the instance of this class using a constructor instead of field-initialization